apps/build-onchain-apps/onchainKit/hooks/useBlockExplorerLink.ts

```diff
@@ -11,7 +11,11 @@ export enum HashType {
  * @param {string} hash - The contract or transaction hash.
  * @returns {string} - The URL to the Etherscan page for the given hash.
  */
-function useBlockExplorerLink(chain: Chain, hash: string, type: HashType = HashType.Address) {
+function useBlockExplorerLink(
```

Comment:
does this really need to be a hook?

Reply:
Oh I see your point, should this just be an utility like `getBlockExplorerLink`?

After all if the chain is changing than someone can just re-run the utility and get a fresh `link`.

Right?

@renanmav thoughts on this?